gammapy/estimators/map/ts.py

```diff
@@ -1042,3 +1076,38 @@ def _ts_value(
         norm_guess=norm_guess,
     )
     return flux_estimator.run(dataset)
+
+
+class BrentqSensitivityEstimator(BrentqFluxEstimator):
+    """Single parameter sensitivity estimator."""
+
+    def _confidence(self, dataset, n_sigma, result, positive):
```

Comment:
As the method is hidden away, just move it to the `BrentqFluxEstimator`? I think the motivation for a  new "BrentqSensitivityEstimator" is weak.

Reply:
yes I removed it.